package.json

```diff
@@ -80,51 +80,61 @@
       {
         "command": "cmake.viewLog",
         "title": "%cmake-tools.command.cmake.viewLog.title%",
+        "when": "!cmake:cmtPartiallyActive",
```

Comment:
Maybe a context name with negated meaning would have saved you adding all of the `!`'s in this file. `cmake.cmtActive` or some such.